cli/module_loader.rs

```diff
@@ -130,7 +133,12 @@ impl ModuleLoader for CliModuleLoader {
     let state = op_state.borrow();
 
     // The permissions that should be applied to any dynamically imported module
-    let dynamic_permissions = state.borrow::<Permissions>().clone();
+    let dynamic_permissions = if self.permissions.is_some() {
+      self.permissions.clone().unwrap()
+    } else {
```

Comment:
```
let dynamic_permissions = if let Some(permissions) = self.permissions.take() {
      self.permissions.clone().unwrap()
    } else {
...
}
```

Reply:
I don't get what you mean by this, why would I need to take out the permissions and left self.permissions empty?